Examples/MAX32650/StaticLibrary/example_lib/example_lib.mk

```diff
@@ -0,0 +1,31 @@
+# Example static library Makefile
+
+# **********************************************************
+# Set the name of your library here. It will be built as CUSTOM_LIB_NAME_softfp.a,
+# LIB_NAME_hardfp.a or just LIB_NAME.a under the project build directory.
+CUSTOM_LIB_NAME=libExample
+# Insert your source files below
+CUSTOM_LIB_SRC_FILES = example.c
+#CUSTOM_LIB_SRC_FILES += example_2.c
+
+# Variables below do not need to be modified in general. 
+CUSTOM_LIB_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+CUSTOM_LIB_OUT_DIR = $(abspath $(CUSTOM_LIB_DIR)/../build)
+CUSTOM_LIB_SRCS = $(addprefix $(CUSTOM_LIB_DIR),$(CUSTOM_LIB_SRC_FILES))
```

Comment:
Can build folder be under library folder? for cuurent name, 
example_lib/build

Also If applicable it will be better if .a file store under example_lib,  not under example/build.

Reply:
"make clean" only deletes the project build folder, it will not clean example_lib/build and the example_lib/*.a. Either clean recipe in the project Makefile has to be modified or the user has to manually delete the library's build output.